azkaban-common/src/main/java/azkaban/project/FlowLoaderUtils.java

```diff
@@ -148,6 +232,33 @@ public static ValidationReport generateFlowLoaderReport(final Set<String> errors
     return report;
   }
 
+  /**
+   * Clean up temp dir.
+   *
+   * @param tempDir the temp dir
+   */
+  public static void cleanUpTempDir(final File tempDir) {
+    logger.info("Cleaning up temp directory.");
```

Comment:
would be great to log the temp dir full path as well.

Reply:
The temp dir path would just be some random string. Any reason why we need to log it?

Reply:
Actually, I think it's ok to delete this log message since it does not provide much useful info.